composer.json

```diff
@@ -16,12 +16,12 @@
         "issues": "https://issues.apache.org/jira/browse/THRIFT"
     },
     "require": {
-        "php": ">=5.3.0"
+        "php": "^5.6 || ^7.0"
```

Comment:
Just looking at the changes here for dropping php 5.3 through 5.5, what's the benefit?  No php code actually changed to take advantage of this, so I'm not sure I see a reason to do it?